frontend/src/views/ManagePermissionsView/utils.ts

```diff
@@ -44,11 +37,7 @@ export const generateAppPermissionSuccessNotifications = (
             .map((filtered) => filtered.detail.role);
 
     const successClientList = successRoleList
-        .map((role) =>
-            isFamRoleWithClientSchema(role)
-                ? role.forest_client
-                : role.client_number
-        )
```

Comment:
Backend has changed to use `role.forest_client` for consistency, so no need to use `role.client_number` anymore (did not make sense for the previous name)